atom/browser/atom_resource_dispatcher_host_delegate.cc

```diff
@@ -75,11 +75,4 @@ bool AtomResourceDispatcherHostDelegate::HandleExternalProtocol(
   return true;
 }
 
-content::ResourceDispatcherHostLoginDelegate*
-AtomResourceDispatcherHostDelegate::CreateLoginDelegate(
```

Comment:
shouldn't we implement `ContentBrowserClient::CreateLoginDelegate`?

Reply:
We don't have very clear this one. We could add an implementation for `ContentBrowserClient::CreateLoginDelegate` as this https://github.com/brave/muon/commit/b32ea1b4a3c459324d5f100b63ce067602a9ec7c, but we are not sure what should be used from `atom/browser/login_handler.h` as we don't have the UI that provides https://chromium.googlesource.com/chromium/src/+/ead4271f098dd2038ec8514f1bf9b59f83b0774b/chrome/browser/ui/login/login_handler.h#139

Reply:
@darkdh was there any resolution on this? Doesn't this break basic auth?

Reply:
verified that basic auth is broken

Reply:
@ltilve the ui portion is not relevant. I have a fix for this and minimal changes were required